docs/metrics.md

```diff
@@ -1,37 +1,140 @@
+  serviceName: 'my-service',
+  // The suggested resource is filled in via OTEL_RESOURCE_ATTRIBUTES
+  resourceFactory: (suggestedResource: Resource) => {
+    return suggestedResource.merge(new Resource({
+      'my.property': 'xyz',
+      'build': 42,
+    }));
+  },
+  exportIntervalMillis: 1000, // default: 5000
+  // The default exporter used is OTLP over gRPC
+  endpoint: 'http://collector:4317',
+});
+
+const meter = metrics.getMeter('my-meter');
+const counter = meter.createCounter('clicks');
+counter.add(3);
+```
+
+## Using custom metric readers and exporters
 
-- `nodejs.memory.heap.total` (gauge, bytes) - Heap total via `process.memoryUsage().heapTotal`.
-- `nodejs.memory.heap.used` (gauge, bytes) - Heap used via `process.memoryUsage().heapUsed`.
-- `nodejs.memory.rss` (gauge, bytes) - Resident set size via `process.memoryUsage().rss`.
-- `nodejs.memory.gc.size` (cumulative_counter, bytes) - Total collected by the garbage collector.
-- `nodejs.memory.gc.pause` (cumulative_counter, nanoseconds) - Time spent doing GC.
-- `nodejs.memory.gc.count` (cumulative_counter, count) - Number of times GC ran.
-- `nodejs.event_loop.lag.max` (gauge, nanoseconds) - Max event loop lag within the collection interval.
-- `nodejs.event_loop.lag.min` (gauge, nanoseconds) - Min event loop lag within the collection interval.
+Custom exporters and/or readers can be provided via the `metricReaderFactory` option.
 
-## Custom metrics
+:warning: Setting `metricReaderFactory` will invalidate `exportInterval` and `endpoint` options.
 
-You can send custom metrics via the internal [SignalFx client](https://github.com/signalfx/signalfx-nodejs#reporting-data):
+```javascript
+const { startMetrics } = require('@splunk/otel');
+const { PrometheusExporter } = require('@opentelemetry/exporter-prometheus');
+const { OTLPMetricExporter } = require('@opentelemetry/exporter-metrics-otlp-http');
+const { PeriodicExportingMetricReader } = require('@opentelemetry/sdk-metrics-base');
+
+startMetrics({
+  serviceName: 'my-service',
+  metricReaderFactory: () => {
+    return [
+      new PrometheusExporter(),
+      new PeriodicExportingMetricReader({
+        exportIntervalMillis: 1000,
+        exporter: new OTLPMetricExporter({ url: 'http://localhost:4318' })
+      })
+    ]
+  }
+});
+```
+
+## Choosing temporality aggregation type
 
 ```javascript
 const { startMetrics } = require('@splunk/otel');
+const { OTLPMetricExporter } = require('@opentelemetry/exporter-metrics-otlp-grpc');
+const { AggregationTemporality, PeriodicExportingMetricReader } = require('@opentelemetry/sdk-metrics-base');
+
+startMetrics({
+  serviceName: 'my-service',
+  metricReaderFactory: () => {
+    return [
+      new PeriodicExportingMetricReader({
+        exporter: new OTLPMetricExporter({
+          temporalityPreference: AggregationTemporality.DELTA
+        })
+      })
+    ]
+  }
+});
+```
+
+## Migrating from SignalFx metrics
+
+The SignalFx client is no longer available.
```

Comment:
I'd rather say that it's deprecated and copy the EoS note.

Reply:
I'm not sure the SignalFx Node.js client is deprecated, but I changed the wording that we removed the dependency on this library.